quipucords/api/tests_report.py

```diff
@@ -44,35 +43,14 @@ def create_fact_collection(self,
         fact_collection.save()
         return fact_collection
 
-    def create_fingerprints(self, fact_collection):
-        """Helper to create a SystemFingerprint for test."""
-
-        fingerprints = []
-        for fact in fact_collection.facts.all():
-            fingerprint = SystemFingerprint \
-                .objects.create(fact_collection_id=fact_collection,
-                                os_name=fact.etc_release_name,
-                                os_release=fact.etc_release_release,
-                                os_version=fact.etc_release_version)
-            fingerprints.append(fingerprint)
-
-        return fingerprints
-
-    def test_fc_creation(self):
-        """ Test model creation not via API."""
-        fingerprints = self.create_fingerprints(self.create_fact_collection())
-        for fingerprint in fingerprints:
-            self.assertTrue(isinstance(fingerprint, SystemFingerprint))
-            self.assertIsInstance(str(fingerprint), str)
-
     def test_get_report_list(self):
         """ Create fact collection object via API."""
 
         url = '/api/v1/reports/'
 
         # Create a system fingerprint
```

Comment:
seems like a stale comment.

Reply:
Its not a stale statement but its also not obvious why it is there.  I will update the comment.